packages/generators/add-generator.ts

```diff
@@ -173,6 +173,36 @@ export default class AddGenerator extends Generator {
 								done();
 							});
 						}
+						if (action === "merge") {
+							const validatePath = (path: string) => {
+								const resolvedPath = resolve(process.env.PWD, path);
+								if (existsSync(resolvedPath)) {
+									if (/\.js$/.test(path)) {
+										if (typeof require(resolvedPath) !== "object") {
+											return "Given file doesn't export an Object";
+										}
+										return true;
+									} else {
+										return "Path doesn't corresponds to a javascript file";
+									}
+								}
+								return "Invalid path provided";
+							};
+							return this.prompt([
+								InputValidate(
+									"mergeFile",
+									"What is the location of webpack configuration file with which you want to merge current configuration?",
```

Comment:
Let's start doing something nice: could you move this long question inside a `constants` file?
Also, the same for the rest of the strings.

Ideally these strings could be exported from `webpack-cli/utils` so they can be reused across the board